Dockerfile

```diff
@@ -224,11 +224,12 @@ FROM ort-base-image AS gobuild
 
 ARG GO_DEP_VERSION=0.5.4
 ARG GO_VERSION=1.18.3
-ENV GOPATH=/opt/go
-RUN curl -L https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /opt -xz
-ENV PATH=/opt/go/bin:$PATH
-RUN go version
-RUN curl -ksS https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | bash
+ENV GOBIN=/opt/go/bin
```

Comment:
commit message: are supported as **these** are the architectures.

Reply:
Fixed